### DISCUSSION TITLE
Microphone access request

### DISCUSSION THREAD

**Discussion by 88Igor88** [rocksdanister/lively#2015] [General]
Why is lively trying to use my microphone?
If I disable access, then the wallpaper is not set, writes an error.
why?

**Comment by rocksdanister** [comment]
One possibility is audio visualizer/Chromium component is probably picking it up when recording and playback device is same etc..

No data is transmitted anywhere, everything is run locally.. you are free to audit the code:

https://github.com/rocksdanister/lively-cef
